pages/k8s/openstack-integration.md

```diff
@@ -24,6 +24,13 @@ The `openstack-integrator` charm simplifies working with **Charmed Kubernetes**
 credentials provided to Juju, it acts as a proxy between Charmed Kubernetes and the underlying cloud,
 granting permissions to dynamically create, for example, Cinder volumes.
 
+### Prerequisites
+
+Openstack integration requires [Octavia][octavia] to be available in the
+underlying OpenStack cloud. Without Octavia, both the openstack-integrator
+charm and the openstack-cloud-controller-manager pod that gets deployed into
+Kubernetes will be unable to create load balancers properly.
```

Comment:
I think I worded this a bit negatively. Let me see if I can make it more positive.